embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/EmbraceImpl.kt

```diff
@@ -269,6 +271,23 @@ internal class EmbraceImpl @JvmOverloads constructor(
         }
     }
 
+    override fun disableDataExport() {
+        if (!sdkCallChecker.started.get()) {
+            return
+        }
+        bootstrapper.openTelemetryModule.openTelemetryConfiguration.disableDataExport()
+        stop()
+
+        // delete any persisted data
+        runCatching {
```

Comment:
Should we do the delete even if the SDK hasn't started yet?

Reply:
Yep, this sounds reasonable.